index.js

```diff
@@ -76,6 +77,20 @@ module.exports = function (str, opts) {
 		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
 	}
 
+	// remove directory index
+	if (opts.removeDirectoryIndex === true) {
+		opts.removeDirectoryIndex = [/^index/];
```

Comment:
@portgasd666 New commit is best as I can easily see what changes you did. I just squash when merging.